core/sr-primitives/src/lib.rs

```diff
@@ -411,11 +421,12 @@ impl Fixed64 {
 
 	/// Performs a saturated multiply and accumulate.
 	///
-	/// Returns `n + (self * n)`.
+	/// Returns a saturated `n + (self * n)`.
+	/// TODO: generalize this to any weight type.
```

Comment:
It is also okay as-is since it is very unlikely for us to decide to change the weight type but still a good +

Reply:
will need an issue number, then.